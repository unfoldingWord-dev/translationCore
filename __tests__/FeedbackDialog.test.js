/* eslint-env jest */
'use strict';
import React from 'react';
import { shallow } from 'enzyme';
import FeedbackDialog from '../src/js/components/dialogComponents/FeedbackDialog';

describe('FeedbackDialog component renders correctly', () => {
  test('FeedbackDialog component render should match snapshot', () => {
    const props = {
      includeLogs: true,
      message: 'Allow miles wound place the leave had. To sitting subject no improve studied limited. Ye indulgence unreserved connection alteration appearance my an astonished. Up as seen sent make he they of. Her raising and himself pasture believe females. Fancy she stuff after aware merit small his. Charmed esteems luckily age out. \n' +
        '\n' +
        'Promotion an ourselves up otherwise my. High what each snug rich far yet easy. In companions inhabiting mr principles at insensible do. Heard their sex hoped enjoy vexed child for. Prosperous so occasional assistance it discovered especially no. Provision of he residence consisted up in remainder arranging described. Conveying has concealed necessary furnished bed zealously immediate get but. Terminated as middletons or by instrument. Bred do four so your felt with. No shameless principle dependent household do. \n' +
        '\n' +
        'Of recommend residence education be on difficult repulsive offending. Judge views had mirth table seems great him for her. Alone all happy asked begin fully stand own get. Excuse ye seeing result of we. See scale dried songs old may not. Promotion did disposing you household any instantly. Hills we do under times at first short an. \n' +
        '\n' +
        'An do on frankness so cordially immediate recommend contained. Imprudence insensible be literature unsatiable do. Of or imprudence solicitude affronting in mr possession. Compass journey he request on suppose limited of or. She margaret law thoughts proposal formerly. Speaking ladyship yet scarcely and mistaken end exertion dwelling. All decisively dispatched instrument particular way one devonshire. Applauded she sportsman explained for out objection. \n',
      email: 'johndoe@acme.com',
      category: 'general_feedback',
      translate: k => k,
      onCLose: jest.fn(),
      onSubmit: jest.fn(),
      open: true,
    };
    const modal = shallow(<FeedbackDialog {...props}/>).dive();
    expect(modal).toMatchSnapshot();
  });
});
