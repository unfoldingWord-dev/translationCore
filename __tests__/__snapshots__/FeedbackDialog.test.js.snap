// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FeedbackDialog component renders correctly FeedbackDialog component render should match snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <BaseDialog
    bodyStyle={
      Object {
        "overflowY": "auto",
      }
    }
    onClose={[Function]}
    onSubmit={[Function]}
    open={true}
    primaryActionEnabled={true}
    primaryLabel="buttons.submit_button"
    secondaryActionEnabled={true}
    secondaryLabel="buttons.cancel_button"
    title="user_feedback_comments"
    titleStyle={Object {}}
  >
    <CategoryPicker
      categories={
        Array [
          Object {
            "key": "General Feedback",
            "value": "general_feedback",
          },
          Object {
            "key": "Content and Resources Feedback",
            "value": "content_resources_feedback",
          },
          Object {
            "key": "Bug Report",
            "value": "bug_report",
          },
        ]
      }
      label="users.category"
      onChange={[Function]}
      selectedCategory="general_feedback"
    />
    <TextField
      autoFocus={true}
      disabled={false}
      errorStyle={
        Object {
          "color": "var(--warning-color)",
        }
      }
      errorText={false}
      floatingLabelFixed={false}
      floatingLabelStyle={
        Object {
          "color": "var(--text-color-dark)",
        }
      }
      floatingLabelText="leave_feedback"
      fullWidth={false}
      multiLine={true}
      onChange={[Function]}
      rows={3}
      style={
        Object {
          "width": "100%",
        }
      }
      type="text"
      underlineFocusStyle={
        Object {
          "borderColor": "var(--accent-color-dark)",
        }
      }
      underlineShow={true}
      value="Allow miles wound place the leave had. To sitting subject no improve studied limited. Ye indulgence unreserved connection alteration appearance my an astonished. Up as seen sent make he they of. Her raising and himself pasture believe females. Fancy she stuff after aware merit small his. Charmed esteems luckily age out. 

Promotion an ourselves up otherwise my. High what each snug rich far yet easy. In companions inhabiting mr principles at insensible do. Heard their sex hoped enjoy vexed child for. Prosperous so occasional assistance it discovered especially no. Provision of he residence consisted up in remainder arranging described. Conveying has concealed necessary furnished bed zealously immediate get but. Terminated as middletons or by instrument. Bred do four so your felt with. No shameless principle dependent household do. 

Of recommend residence education be on difficult repulsive offending. Judge views had mirth table seems great him for her. Alone all happy asked begin fully stand own get. Excuse ye seeing result of we. See scale dried songs old may not. Promotion did disposing you household any instantly. Hills we do under times at first short an. 

An do on frankness so cordially immediate recommend contained. Imprudence insensible be literature unsatiable do. Of or imprudence solicitude affronting in mr possession. Compass journey he request on suppose limited of or. She margaret law thoughts proposal formerly. Speaking ladyship yet scarcely and mistaken end exertion dwelling. All decisively dispatched instrument particular way one devonshire. Applauded she sportsman explained for out objection. 
"
    />
    <TextField
      disabled={false}
      errorStyle={
        Object {
          "color": "var(--warning-color)",
        }
      }
      errorText={false}
      floatingLabelFixed={false}
      floatingLabelStyle={
        Object {
          "color": "var(--text-color-dark)",
        }
      }
      floatingLabelText="email_address_optional"
      fullWidth={false}
      multiLine={false}
      onChange={[Function]}
      rows={1}
      style={
        Object {
          "width": "100%",
        }
      }
      type="text"
      underlineFocusStyle={
        Object {
          "borderColor": "var(--accent-color-dark)",
        }
      }
      underlineShow={true}
      value="johndoe@acme.com"
    />
    <Checkbox
      checked={true}
      disabled={false}
      iconStyle={
        Object {
          "fill": "var(--accent-color-dark)",
        }
      }
      label="include_app_logs"
      labelPosition="right"
      onCheck={[Function]}
      style={
        Object {
          "marginTop": "15px",
        }
      }
    />
  </BaseDialog>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": <Dialog
        actions={
          Array [
            <button
              className="btn-second"
              disabled={false}
              onClick={[Function]}
            >
              buttons.cancel_button
            </button>,
            <button
              className="btn-prime"
              disabled={false}
              onClick={[Function]}
            >
              buttons.submit_button
            </button>,
          ]
        }
        autoDetectWindowHeight={true}
        autoScrollBodyContent={false}
        bodyStyle={
          Object {
            "overflowY": "auto",
          }
        }
        modal={true}
        onRequestClose={[Function]}
        open={true}
        repositionOnUpdate={true}
        title="user_feedback_comments"
        titleStyle={
          Object {
            "backgroundColor": "var(--accent-color-dark)",
            "color": "var(--reverse-color)",
            "marginBottom": "15px",
            "padding": "15px",
          }
        }
      >
        <CategoryPicker
          categories={
            Array [
              Object {
                "key": "General Feedback",
                "value": "general_feedback",
              },
              Object {
                "key": "Content and Resources Feedback",
                "value": "content_resources_feedback",
              },
              Object {
                "key": "Bug Report",
                "value": "bug_report",
              },
            ]
          }
          label="users.category"
          onChange={[Function]}
          selectedCategory="general_feedback"
        />
        <TextField
          autoFocus={true}
          disabled={false}
          errorStyle={
            Object {
              "color": "var(--warning-color)",
            }
          }
          errorText={false}
          floatingLabelFixed={false}
          floatingLabelStyle={
            Object {
              "color": "var(--text-color-dark)",
            }
          }
          floatingLabelText="leave_feedback"
          fullWidth={false}
          multiLine={true}
          onChange={[Function]}
          rows={3}
          style={
            Object {
              "width": "100%",
            }
          }
          type="text"
          underlineFocusStyle={
            Object {
              "borderColor": "var(--accent-color-dark)",
            }
          }
          underlineShow={true}
          value="Allow miles wound place the leave had. To sitting subject no improve studied limited. Ye indulgence unreserved connection alteration appearance my an astonished. Up as seen sent make he they of. Her raising and himself pasture believe females. Fancy she stuff after aware merit small his. Charmed esteems luckily age out. 

Promotion an ourselves up otherwise my. High what each snug rich far yet easy. In companions inhabiting mr principles at insensible do. Heard their sex hoped enjoy vexed child for. Prosperous so occasional assistance it discovered especially no. Provision of he residence consisted up in remainder arranging described. Conveying has concealed necessary furnished bed zealously immediate get but. Terminated as middletons or by instrument. Bred do four so your felt with. No shameless principle dependent household do. 

Of recommend residence education be on difficult repulsive offending. Judge views had mirth table seems great him for her. Alone all happy asked begin fully stand own get. Excuse ye seeing result of we. See scale dried songs old may not. Promotion did disposing you household any instantly. Hills we do under times at first short an. 

An do on frankness so cordially immediate recommend contained. Imprudence insensible be literature unsatiable do. Of or imprudence solicitude affronting in mr possession. Compass journey he request on suppose limited of or. She margaret law thoughts proposal formerly. Speaking ladyship yet scarcely and mistaken end exertion dwelling. All decisively dispatched instrument particular way one devonshire. Applauded she sportsman explained for out objection. 
"
        />
        <TextField
          disabled={false}
          errorStyle={
            Object {
              "color": "var(--warning-color)",
            }
          }
          errorText={false}
          floatingLabelFixed={false}
          floatingLabelStyle={
            Object {
              "color": "var(--text-color-dark)",
            }
          }
          floatingLabelText="email_address_optional"
          fullWidth={false}
          multiLine={false}
          onChange={[Function]}
          rows={1}
          style={
            Object {
              "width": "100%",
            }
          }
          type="text"
          underlineFocusStyle={
            Object {
              "borderColor": "var(--accent-color-dark)",
            }
          }
          underlineShow={true}
          value="johndoe@acme.com"
        />
        <Checkbox
          checked={true}
          disabled={false}
          iconStyle={
            Object {
              "fill": "var(--accent-color-dark)",
            }
          }
          label="include_app_logs"
          labelPosition="right"
          onCheck={[Function]}
          style={
            Object {
              "marginTop": "15px",
            }
          }
        />
      </Dialog>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "actions": Array [
          <button
            className="btn-second"
            disabled={false}
            onClick={[Function]}
          >
            buttons.cancel_button
          </button>,
          <button
            className="btn-prime"
            disabled={false}
            onClick={[Function]}
          >
            buttons.submit_button
          </button>,
        ],
        "autoDetectWindowHeight": true,
        "autoScrollBodyContent": false,
        "bodyStyle": Object {
          "overflowY": "auto",
        },
        "children": Array [
          <CategoryPicker
            categories={
              Array [
                Object {
                  "key": "General Feedback",
                  "value": "general_feedback",
                },
                Object {
                  "key": "Content and Resources Feedback",
                  "value": "content_resources_feedback",
                },
                Object {
                  "key": "Bug Report",
                  "value": "bug_report",
                },
              ]
            }
            label="users.category"
            onChange={[Function]}
            selectedCategory="general_feedback"
          />,
          <TextField
            autoFocus={true}
            disabled={false}
            errorStyle={
              Object {
                "color": "var(--warning-color)",
              }
            }
            errorText={false}
            floatingLabelFixed={false}
            floatingLabelStyle={
              Object {
                "color": "var(--text-color-dark)",
              }
            }
            floatingLabelText="leave_feedback"
            fullWidth={false}
            multiLine={true}
            onChange={[Function]}
            rows={3}
            style={
              Object {
                "width": "100%",
              }
            }
            type="text"
            underlineFocusStyle={
              Object {
                "borderColor": "var(--accent-color-dark)",
              }
            }
            underlineShow={true}
            value="Allow miles wound place the leave had. To sitting subject no improve studied limited. Ye indulgence unreserved connection alteration appearance my an astonished. Up as seen sent make he they of. Her raising and himself pasture believe females. Fancy she stuff after aware merit small his. Charmed esteems luckily age out. 

Promotion an ourselves up otherwise my. High what each snug rich far yet easy. In companions inhabiting mr principles at insensible do. Heard their sex hoped enjoy vexed child for. Prosperous so occasional assistance it discovered especially no. Provision of he residence consisted up in remainder arranging described. Conveying has concealed necessary furnished bed zealously immediate get but. Terminated as middletons or by instrument. Bred do four so your felt with. No shameless principle dependent household do. 

Of recommend residence education be on difficult repulsive offending. Judge views had mirth table seems great him for her. Alone all happy asked begin fully stand own get. Excuse ye seeing result of we. See scale dried songs old may not. Promotion did disposing you household any instantly. Hills we do under times at first short an. 

An do on frankness so cordially immediate recommend contained. Imprudence insensible be literature unsatiable do. Of or imprudence solicitude affronting in mr possession. Compass journey he request on suppose limited of or. She margaret law thoughts proposal formerly. Speaking ladyship yet scarcely and mistaken end exertion dwelling. All decisively dispatched instrument particular way one devonshire. Applauded she sportsman explained for out objection. 
"
          />,
          <TextField
            disabled={false}
            errorStyle={
              Object {
                "color": "var(--warning-color)",
              }
            }
            errorText={false}
            floatingLabelFixed={false}
            floatingLabelStyle={
              Object {
                "color": "var(--text-color-dark)",
              }
            }
            floatingLabelText="email_address_optional"
            fullWidth={false}
            multiLine={false}
            onChange={[Function]}
            rows={1}
            style={
              Object {
                "width": "100%",
              }
            }
            type="text"
            underlineFocusStyle={
              Object {
                "borderColor": "var(--accent-color-dark)",
              }
            }
            underlineShow={true}
            value="johndoe@acme.com"
          />,
          <Checkbox
            checked={true}
            disabled={false}
            iconStyle={
              Object {
                "fill": "var(--accent-color-dark)",
              }
            }
            label="include_app_logs"
            labelPosition="right"
            onCheck={[Function]}
            style={
              Object {
                "marginTop": "15px",
              }
            }
          />,
        ],
        "modal": true,
        "onRequestClose": [Function],
        "open": true,
        "repositionOnUpdate": true,
        "title": "user_feedback_comments",
        "titleStyle": Object {
          "backgroundColor": "var(--accent-color-dark)",
          "color": "var(--reverse-color)",
          "marginBottom": "15px",
          "padding": "15px",
        },
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "categories": Array [
              Object {
                "key": "General Feedback",
                "value": "general_feedback",
              },
              Object {
                "key": "Content and Resources Feedback",
                "value": "content_resources_feedback",
              },
              Object {
                "key": "Bug Report",
                "value": "bug_report",
              },
            ],
            "label": "users.category",
            "onChange": [Function],
            "selectedCategory": "general_feedback",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "autoFocus": true,
            "disabled": false,
            "errorStyle": Object {
              "color": "var(--warning-color)",
            },
            "errorText": false,
            "floatingLabelFixed": false,
            "floatingLabelStyle": Object {
              "color": "var(--text-color-dark)",
            },
            "floatingLabelText": "leave_feedback",
            "fullWidth": false,
            "multiLine": true,
            "onChange": [Function],
            "rows": 3,
            "style": Object {
              "width": "100%",
            },
            "type": "text",
            "underlineFocusStyle": Object {
              "borderColor": "var(--accent-color-dark)",
            },
            "underlineShow": true,
            "value": "Allow miles wound place the leave had. To sitting subject no improve studied limited. Ye indulgence unreserved connection alteration appearance my an astonished. Up as seen sent make he they of. Her raising and himself pasture believe females. Fancy she stuff after aware merit small his. Charmed esteems luckily age out. 

Promotion an ourselves up otherwise my. High what each snug rich far yet easy. In companions inhabiting mr principles at insensible do. Heard their sex hoped enjoy vexed child for. Prosperous so occasional assistance it discovered especially no. Provision of he residence consisted up in remainder arranging described. Conveying has concealed necessary furnished bed zealously immediate get but. Terminated as middletons or by instrument. Bred do four so your felt with. No shameless principle dependent household do. 

Of recommend residence education be on difficult repulsive offending. Judge views had mirth table seems great him for her. Alone all happy asked begin fully stand own get. Excuse ye seeing result of we. See scale dried songs old may not. Promotion did disposing you household any instantly. Hills we do under times at first short an. 

An do on frankness so cordially immediate recommend contained. Imprudence insensible be literature unsatiable do. Of or imprudence solicitude affronting in mr possession. Compass journey he request on suppose limited of or. She margaret law thoughts proposal formerly. Speaking ladyship yet scarcely and mistaken end exertion dwelling. All decisively dispatched instrument particular way one devonshire. Applauded she sportsman explained for out objection. 
",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "disabled": false,
            "errorStyle": Object {
              "color": "var(--warning-color)",
            },
            "errorText": false,
            "floatingLabelFixed": false,
            "floatingLabelStyle": Object {
              "color": "var(--text-color-dark)",
            },
            "floatingLabelText": "email_address_optional",
            "fullWidth": false,
            "multiLine": false,
            "onChange": [Function],
            "rows": 1,
            "style": Object {
              "width": "100%",
            },
            "type": "text",
            "underlineFocusStyle": Object {
              "borderColor": "var(--accent-color-dark)",
            },
            "underlineShow": true,
            "value": "johndoe@acme.com",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "checked": true,
            "disabled": false,
            "iconStyle": Object {
              "fill": "var(--accent-color-dark)",
            },
            "label": "include_app_logs",
            "labelPosition": "right",
            "onCheck": [Function],
            "style": Object {
              "marginTop": "15px",
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": [Function],
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <Dialog
          actions={
            Array [
              <button
                className="btn-second"
                disabled={false}
                onClick={[Function]}
              >
                buttons.cancel_button
              </button>,
              <button
                className="btn-prime"
                disabled={false}
                onClick={[Function]}
              >
                buttons.submit_button
              </button>,
            ]
          }
          autoDetectWindowHeight={true}
          autoScrollBodyContent={false}
          bodyStyle={
            Object {
              "overflowY": "auto",
            }
          }
          modal={true}
          onRequestClose={[Function]}
          open={true}
          repositionOnUpdate={true}
          title="user_feedback_comments"
          titleStyle={
            Object {
              "backgroundColor": "var(--accent-color-dark)",
              "color": "var(--reverse-color)",
              "marginBottom": "15px",
              "padding": "15px",
            }
          }
        >
          <CategoryPicker
            categories={
              Array [
                Object {
                  "key": "General Feedback",
                  "value": "general_feedback",
                },
                Object {
                  "key": "Content and Resources Feedback",
                  "value": "content_resources_feedback",
                },
                Object {
                  "key": "Bug Report",
                  "value": "bug_report",
                },
              ]
            }
            label="users.category"
            onChange={[Function]}
            selectedCategory="general_feedback"
          />
          <TextField
            autoFocus={true}
            disabled={false}
            errorStyle={
              Object {
                "color": "var(--warning-color)",
              }
            }
            errorText={false}
            floatingLabelFixed={false}
            floatingLabelStyle={
              Object {
                "color": "var(--text-color-dark)",
              }
            }
            floatingLabelText="leave_feedback"
            fullWidth={false}
            multiLine={true}
            onChange={[Function]}
            rows={3}
            style={
              Object {
                "width": "100%",
              }
            }
            type="text"
            underlineFocusStyle={
              Object {
                "borderColor": "var(--accent-color-dark)",
              }
            }
            underlineShow={true}
            value="Allow miles wound place the leave had. To sitting subject no improve studied limited. Ye indulgence unreserved connection alteration appearance my an astonished. Up as seen sent make he they of. Her raising and himself pasture believe females. Fancy she stuff after aware merit small his. Charmed esteems luckily age out. 

Promotion an ourselves up otherwise my. High what each snug rich far yet easy. In companions inhabiting mr principles at insensible do. Heard their sex hoped enjoy vexed child for. Prosperous so occasional assistance it discovered especially no. Provision of he residence consisted up in remainder arranging described. Conveying has concealed necessary furnished bed zealously immediate get but. Terminated as middletons or by instrument. Bred do four so your felt with. No shameless principle dependent household do. 

Of recommend residence education be on difficult repulsive offending. Judge views had mirth table seems great him for her. Alone all happy asked begin fully stand own get. Excuse ye seeing result of we. See scale dried songs old may not. Promotion did disposing you household any instantly. Hills we do under times at first short an. 

An do on frankness so cordially immediate recommend contained. Imprudence insensible be literature unsatiable do. Of or imprudence solicitude affronting in mr possession. Compass journey he request on suppose limited of or. She margaret law thoughts proposal formerly. Speaking ladyship yet scarcely and mistaken end exertion dwelling. All decisively dispatched instrument particular way one devonshire. Applauded she sportsman explained for out objection. 
"
          />
          <TextField
            disabled={false}
            errorStyle={
              Object {
                "color": "var(--warning-color)",
              }
            }
            errorText={false}
            floatingLabelFixed={false}
            floatingLabelStyle={
              Object {
                "color": "var(--text-color-dark)",
              }
            }
            floatingLabelText="email_address_optional"
            fullWidth={false}
            multiLine={false}
            onChange={[Function]}
            rows={1}
            style={
              Object {
                "width": "100%",
              }
            }
            type="text"
            underlineFocusStyle={
              Object {
                "borderColor": "var(--accent-color-dark)",
              }
            }
            underlineShow={true}
            value="johndoe@acme.com"
          />
          <Checkbox
            checked={true}
            disabled={false}
            iconStyle={
              Object {
                "fill": "var(--accent-color-dark)",
              }
            }
            label="include_app_logs"
            labelPosition="right"
            onCheck={[Function]}
            style={
              Object {
                "marginTop": "15px",
              }
            }
          />
        </Dialog>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "actions": Array [
            <button
              className="btn-second"
              disabled={false}
              onClick={[Function]}
            >
              buttons.cancel_button
            </button>,
            <button
              className="btn-prime"
              disabled={false}
              onClick={[Function]}
            >
              buttons.submit_button
            </button>,
          ],
          "autoDetectWindowHeight": true,
          "autoScrollBodyContent": false,
          "bodyStyle": Object {
            "overflowY": "auto",
          },
          "children": Array [
            <CategoryPicker
              categories={
                Array [
                  Object {
                    "key": "General Feedback",
                    "value": "general_feedback",
                  },
                  Object {
                    "key": "Content and Resources Feedback",
                    "value": "content_resources_feedback",
                  },
                  Object {
                    "key": "Bug Report",
                    "value": "bug_report",
                  },
                ]
              }
              label="users.category"
              onChange={[Function]}
              selectedCategory="general_feedback"
            />,
            <TextField
              autoFocus={true}
              disabled={false}
              errorStyle={
                Object {
                  "color": "var(--warning-color)",
                }
              }
              errorText={false}
              floatingLabelFixed={false}
              floatingLabelStyle={
                Object {
                  "color": "var(--text-color-dark)",
                }
              }
              floatingLabelText="leave_feedback"
              fullWidth={false}
              multiLine={true}
              onChange={[Function]}
              rows={3}
              style={
                Object {
                  "width": "100%",
                }
              }
              type="text"
              underlineFocusStyle={
                Object {
                  "borderColor": "var(--accent-color-dark)",
                }
              }
              underlineShow={true}
              value="Allow miles wound place the leave had. To sitting subject no improve studied limited. Ye indulgence unreserved connection alteration appearance my an astonished. Up as seen sent make he they of. Her raising and himself pasture believe females. Fancy she stuff after aware merit small his. Charmed esteems luckily age out. 

Promotion an ourselves up otherwise my. High what each snug rich far yet easy. In companions inhabiting mr principles at insensible do. Heard their sex hoped enjoy vexed child for. Prosperous so occasional assistance it discovered especially no. Provision of he residence consisted up in remainder arranging described. Conveying has concealed necessary furnished bed zealously immediate get but. Terminated as middletons or by instrument. Bred do four so your felt with. No shameless principle dependent household do. 

Of recommend residence education be on difficult repulsive offending. Judge views had mirth table seems great him for her. Alone all happy asked begin fully stand own get. Excuse ye seeing result of we. See scale dried songs old may not. Promotion did disposing you household any instantly. Hills we do under times at first short an. 

An do on frankness so cordially immediate recommend contained. Imprudence insensible be literature unsatiable do. Of or imprudence solicitude affronting in mr possession. Compass journey he request on suppose limited of or. She margaret law thoughts proposal formerly. Speaking ladyship yet scarcely and mistaken end exertion dwelling. All decisively dispatched instrument particular way one devonshire. Applauded she sportsman explained for out objection. 
"
            />,
            <TextField
              disabled={false}
              errorStyle={
                Object {
                  "color": "var(--warning-color)",
                }
              }
              errorText={false}
              floatingLabelFixed={false}
              floatingLabelStyle={
                Object {
                  "color": "var(--text-color-dark)",
                }
              }
              floatingLabelText="email_address_optional"
              fullWidth={false}
              multiLine={false}
              onChange={[Function]}
              rows={1}
              style={
                Object {
                  "width": "100%",
                }
              }
              type="text"
              underlineFocusStyle={
                Object {
                  "borderColor": "var(--accent-color-dark)",
                }
              }
              underlineShow={true}
              value="johndoe@acme.com"
            />,
            <Checkbox
              checked={true}
              disabled={false}
              iconStyle={
                Object {
                  "fill": "var(--accent-color-dark)",
                }
              }
              label="include_app_logs"
              labelPosition="right"
              onCheck={[Function]}
              style={
                Object {
                  "marginTop": "15px",
                }
              }
            />,
          ],
          "modal": true,
          "onRequestClose": [Function],
          "open": true,
          "repositionOnUpdate": true,
          "title": "user_feedback_comments",
          "titleStyle": Object {
            "backgroundColor": "var(--accent-color-dark)",
            "color": "var(--reverse-color)",
            "marginBottom": "15px",
            "padding": "15px",
          },
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "categories": Array [
                Object {
                  "key": "General Feedback",
                  "value": "general_feedback",
                },
                Object {
                  "key": "Content and Resources Feedback",
                  "value": "content_resources_feedback",
                },
                Object {
                  "key": "Bug Report",
                  "value": "bug_report",
                },
              ],
              "label": "users.category",
              "onChange": [Function],
              "selectedCategory": "general_feedback",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "autoFocus": true,
              "disabled": false,
              "errorStyle": Object {
                "color": "var(--warning-color)",
              },
              "errorText": false,
              "floatingLabelFixed": false,
              "floatingLabelStyle": Object {
                "color": "var(--text-color-dark)",
              },
              "floatingLabelText": "leave_feedback",
              "fullWidth": false,
              "multiLine": true,
              "onChange": [Function],
              "rows": 3,
              "style": Object {
                "width": "100%",
              },
              "type": "text",
              "underlineFocusStyle": Object {
                "borderColor": "var(--accent-color-dark)",
              },
              "underlineShow": true,
              "value": "Allow miles wound place the leave had. To sitting subject no improve studied limited. Ye indulgence unreserved connection alteration appearance my an astonished. Up as seen sent make he they of. Her raising and himself pasture believe females. Fancy she stuff after aware merit small his. Charmed esteems luckily age out. 

Promotion an ourselves up otherwise my. High what each snug rich far yet easy. In companions inhabiting mr principles at insensible do. Heard their sex hoped enjoy vexed child for. Prosperous so occasional assistance it discovered especially no. Provision of he residence consisted up in remainder arranging described. Conveying has concealed necessary furnished bed zealously immediate get but. Terminated as middletons or by instrument. Bred do four so your felt with. No shameless principle dependent household do. 

Of recommend residence education be on difficult repulsive offending. Judge views had mirth table seems great him for her. Alone all happy asked begin fully stand own get. Excuse ye seeing result of we. See scale dried songs old may not. Promotion did disposing you household any instantly. Hills we do under times at first short an. 

An do on frankness so cordially immediate recommend contained. Imprudence insensible be literature unsatiable do. Of or imprudence solicitude affronting in mr possession. Compass journey he request on suppose limited of or. She margaret law thoughts proposal formerly. Speaking ladyship yet scarcely and mistaken end exertion dwelling. All decisively dispatched instrument particular way one devonshire. Applauded she sportsman explained for out objection. 
",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "disabled": false,
              "errorStyle": Object {
                "color": "var(--warning-color)",
              },
              "errorText": false,
              "floatingLabelFixed": false,
              "floatingLabelStyle": Object {
                "color": "var(--text-color-dark)",
              },
              "floatingLabelText": "email_address_optional",
              "fullWidth": false,
              "multiLine": false,
              "onChange": [Function],
              "rows": 1,
              "style": Object {
                "width": "100%",
              },
              "type": "text",
              "underlineFocusStyle": Object {
                "borderColor": "var(--accent-color-dark)",
              },
              "underlineShow": true,
              "value": "johndoe@acme.com",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "checked": true,
              "disabled": false,
              "iconStyle": Object {
                "fill": "var(--accent-color-dark)",
              },
              "label": "include_app_logs",
              "labelPosition": "right",
              "onCheck": [Function],
              "style": Object {
                "marginTop": "15px",
              },
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
        ],
        "type": [Function],
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "context": Object {},
    Symbol(enzyme.__providerValues__): Map {},
  },
  Symbol(enzyme.__providerValues__): Map {},
  Symbol(enzyme.__childContext__): null,
}
`;
